Neos.Flow/Tests/Unit/Persistence/Doctrine/PersistenceManagerTest.php

```diff
@@ -85,14 +83,16 @@ protected function setUp(): void
 
         $allowedObjectsContainer = new AllowedObjectsContainer();
         $this->inject($this->persistenceManager, 'allowedObjects', $allowedObjectsContainer);
-        $allowedObjectsListener = new AllowedObjectsListener();
+        $allowedObjectsListener = $this->getMockBuilder(AllowedObjectsListener::class)->setMethods(['ping'])->getMock();
```

Comment:
Wait, why does the `AllowedObjectsListener` get a `ping` method?

Reply:
Have a look at the code base, I don't know why, but this moved to the `AllowedObjectsListener` a while ago and this here never got updated, I wonder why it didn't fail before.

Reply:
Ok see more detail in the other comment above. So it happened at the start of this whole PR because connectino no longer has ping so we need our own. I am not necessarily sold on it being in the AllowedObjectsListener but I guess it's as good a place as any?

Reply:
> Looking through the history of this change, https://github.com/neos/flow-development-collection/commit/b17b9f0333499e2299700d4007b9f0d69312887c added it to the AllowedObjectsListener because it didn'T exist on the connection anymore